src/app/models/planets.spec.ts

```diff
@@ -1,4 +1,4 @@
-import { Planets } from './planets';
+import { Planets } from './Planets';
```

Comment:
The class naming convention doesn't apply directly to filenames. This is why we've been able to use things like `counter-list.componenet.ts` and inside the file the class is named `CounterListComponent`.

Reply:
Do you mean that the file path name for Planets shouldn't be capitalized even though the file names are capitalized?

Reply:
Correct. The file names can remain the same way we've used them so far.